core/storage/email/gae_models.py

```diff
@@ -352,13 +362,15 @@ class BulkEmailModel(base_models.BaseModel):
     sent_datetime = (
         datastore_services.DateTimeProperty(required=True, indexed=True))
 
+    # DEPRECATED in v3.2.1. Do not use.
+    recipient_ids = datastore_services.JsonProperty(default=[], compressed=True)
```

Comment:
Maybe should leave some comments in the BulkEmailModel docstring to explain why we don't store recipient_ids here and how to store them instead.

Also one question. Don't we need the list of recipient IDs to actually send the emails?

Reply:
Added the docstring.

Yep, but they are only required when we send the emails, this model is only created after that.